manifests/repo.pp

```diff
@@ -93,7 +88,10 @@
           }
         } else {
           $operatingsystem = downcase($::operatingsystem)
-
+          case $::operatingsystemrelease {
+            /\/sid$/ : { $releasename = regsubst($::operatingsystemrelease, '/sid$', '') }
+            default  : { $releasename = $::lsbdistcodename }
```

Comment:
Is lsbdistcodename a thing on newer facter? I seem to recall the lsb* facts being deprecated.